lib/matplotlib/widgets.py

```diff
@@ -3615,6 +3636,73 @@ def __init__(self, ax, onselect, useblit=False,
         self.grab_range = grab_range
 
         self.set_visible(True)
+        self._draw_box = draw_box
+        self._box = None
+        self._box_handle_props = (
+            self._handle_props.update(box_handle_props) or None)
+        self._box_props = box_props or None
```

Comment:
Also, this falls back to `None`, which would trigger for `{}`, so why not default to `None` in the signature?